WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -256,4 +301,14 @@ private Person getCurrentPerson() {
 
         return personDetailFragment.loadPerson();
     }
+
+    @Override
+    public void onFetchMorePeople() {
+        if (mPeopleEndOfListReached) {
+            return;
+        }
+        Blog blog = WordPress.getCurrentBlog();
```

Comment:
Since this only needs the blog's ID, it could use `WordPress.getCurrentLocalTableBlogId()`